scripts/qa_crowbarsetup.sh

```diff
@@ -2335,7 +2335,10 @@ function oncontroller_testsetup()
         crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
         pushd /var/lib/openstack-tempest-test
         echo 1 > /proc/sys/kernel/sysrq
-        ./run_tempest.sh -N $tempestoptions 2>&1 | tee tempest.log
+        if [ ! -d .testrepository ]; then
+            testr init
+        fi
+        testr run $tempestoptions | tee tempest.log | subunit-2to1 | tools/colorizer.py
```

Comment:
I do not understand this.  As you can see in [1], the first version of `run_tempest.sh` is actually using `testr` to run tempest. This script is only a wrapper to `testr init` / `testr run`.

Also, I am not breaking the tempest output, is still there because we will use in Jenkins the output from tools/colorizer.py, that will be exactly the same thing that we have now.

I do not feel confident that to run the tempests tests in `mkcloud` I need to patch the system run_tempest.sh.

[1] https://github.com/openstack/tempest/commit/61f7d5ece5fa5f237ff692f585f5f8e0a6448bb5#diff-33d4e96608fd2cc7b5413e4eb3f48fce